.drone.yml

```diff
@@ -25,7 +25,7 @@ pipeline:
     image: plugins/docker:17.05
     pull: true
     repo: plugins/git
-    tags: [ latest, 1.3.0, 1.3, 1 ]
+    tags: [ latest, 1.4.0, 1.4, 1.3.0, 1.3, 1 ]
```

Comment:
thanks, can you remove the 1.3 tags? then we are good to merge :)

```diff
-tags: [ latest, 1.4.0, 1.4, 1.3.0, 1.3, 1 ]
+tags: [ latest, 1.4.0, 1.4, 1 ]
```

Reply:
Ohhhh, I see. Sorry, pushing it now.